### PR TITLE
Car Insurance Prediction

### PR DESCRIPTION
**Pull request by soma2000-lang**
## Related Issue 

- Info about the related issue 

- [x] LGM-SOC'21 Participant 
- [x] Contributor

Closes: #463

### Descrition

PROJECT TITLE : Car insurance prediction

GOAL : To predict the insurance  in cars
DATASET : 
WHAT I HAD DONE : In this project first I performed a exploratory data analysis on the  dataset which includes of data cleaning , data manipulation, data preprocessing , data visualization and after that I did the model building using  machine learning classification and regression algorithms and then predicted the accuracy of  the model . In the model prediction part I used different machine learning algorithms  . In the EDA part I have included different plots for the different visualizations of our dataset . During the model prediction I got different accuracies from different models , I got the highest accuracy of 99 % using the Random Forest Regressor which is quite well for the given Brazil Fires dataset . Some plots which I used for visualizing the dataset are Histogram., Boxplot, Heatmap etc.

LIBRARIES:

PANDAS

NUMPY

MATPLOTLIB

TENSORFLOW

SKLEARN

CONCLUSION :

So we get a good accuracy and training score of about 99% using Random Forest Regressor

## Type of change

What sort of change have you made:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.

### CODE REVIEW

**Comment by prathimacode-hub** [comment]
I guess last few sections of your project file aren't compiled yet? @soma2000-lang

**Comment by soma2000-lang** [comment]
@prathimacode-hub  is the rest of the part ok

**Comment by prathimacode-hub** [comment]
> @prathimacode-hub is the rest of the part ok

Yes, rest is done well. @soma2000-lang

**Comment by soma2000-lang** [comment]
So can u please merge it

**Comment by prathimacode-hub** [comment]
> So can u please merge it @prathimacode-hub mam

Ok, I would. @soma2000-lang

**Review by prathimacode-hub — APPROVED** [review]
Amazing work. Wonderful visualizations. You had analyzed the data clearly. Yayy. Congrats on your first PR. Welcome to the repo officially. Keep rising high. @soma2000-lang
